app/src/main/java/org/fossasia/phimpme/accounts/AccountActivity.java

```diff
@@ -102,6 +101,7 @@
     private static final int OWNCLOUD_REQUEST_CODE = 9;
     private static final int RESULT_OK = 1;
     private static final int RC_SIGN_IN = 9001;
+    public final static String CLOUDRAIL_APP_KEY = "59c956f8c8b6755b444e4595";
```

Comment:
@codedsun Please remove your API key

Reply:
Ok !Updating ! KIndly make the cloudrail account :)

Reply:
Kindly remove your app key so that I can merge this.